example_scripts/example_runScripts_mantid.py

```diff
@@ -22,6 +22,7 @@
 fitbenchmarking_folder = os.path.abspath(os.path.join(current_path, os.pardir))
 scripts_folder = os.path.join(fitbenchmarking_folder, 'fitbenchmarking')
 sys.path.insert(0, scripts_folder)
+sys.path.insert(1, fitbenchmarking_folder)
```

Comment:
question: is this needed if setup.py has been run?

Reply:
This has been removed in another PR now

Reply:
This has been removed in other PR